crates/sel4-capdl-initializer/core/src/lib.rs

```diff
@@ -359,6 +359,28 @@ impl<'a, N: ObjectName, D: Content, M: GetEmbeddedFrame, B: BorrowMut<[PerObject
                                 }
                             }
                         }
+                        #[sel4_cfg(any(ARCH_X86))]
+                        Object::IrqMsi(obj) => {
```

Comment:
I really dislike that `IrqMsi` etc naming (it should really be something like `x86IrqMsi`) but wanted to be consistent with the CapDL tool's naming.

Reply:
Let me know what you think I should do.